napalm_yang/mappings/eos/parsers/state/openconfig-platform/components.yaml

```diff
@@ -0,0 +1,75 @@
+---
+metadata:
+    parser: JSONParser
+    execute:
+        - method: "device.run_commands"
+          args:
+              commands: ["show inventory"]
+
+components:
+    _parse: unnecessary
+    component:
+        _parse:
+            mode: dict
+            from: "{%- set data = {} -%}{%- for k, v in bookmarks.components.0.0.items() -%}{%- if v is mapping and k is not equalto 'systemInformation' -%}{%- for kk, vv in v.items() -%}{%- do data.update({'{}_{}'.format(k,kk): vv}) -%} {%- endfor -%} {%- endif -%} {%- endfor -%} {{ data|json }}"
```

Comment:
@dbarrosop commented in slack that this might be better implemented as a custom filter, which I had not thought of.  Should we standardize device output transformation through filters?